vignettes/template-doc.Rmd

```diff
@@ -19,15 +19,15 @@ will find the following assets for recording your projects metadata:
 
 * methods.docx
 * abstract.docx
-* metadata.xlsx
+* project-metadata.xlsx
```

Comment:
Update metadata template materials in the aws bucket (linked above as metadata_template materials) 
You can just update the existing docs in there with the up-to-date metadata template example and text. There is a pdf metadata template documentation in there that is generated by running this vignette but saving as a PDF (update output section of md frontmatter). Let me know if you have questions on this.

Reply:
i added the updated metadata template.zip (including the updated pdf) to the aws bucket so it should be linking correctly from the site!